uiowa/brand-icon-browser
# New icons, December 2022

Preview link:
https://sandbox.imu.uiowa.edu

## How to test:
- Use the preview link above or pull down this branch and build it locally with `npm install`, `npm run build` (takes around ten minutes to build) followed by `npm run serve`.
- Test/check for the existence of the icons and their categories described in the next section.
- Test/check the adjusted search terms/keywords in the "Icon adjustments" section.

## Icon updates

This PR updates the [uiowa-brand-icons](https://github.com/uiowa/brand-icons) dependency, which includes the following updates:

## 35 brand new brand icons! 

<img width="1290" alt="Screenshot 2022-12-07 at 11 25 52 AM" src="https://user-images.githubusercontent.com/472923/206248493-862a709d-c7ef-44d5-8672-b58251fe09a0.png">

- `ballot`
- `band-hat`
- `bass-drum`
- `benefits`
- `bionic-bus`
- `clarinet`
- `conducting`
- `drum-major`
- `emotions`
- `empty-battery`
- `fast-forward`
- `full-battery`
- `half-battery`
- `idea`
- `nurse-cap`
- `pappajohn`
- `pause`
- `person`
- `piano`
- `play`
- `quotes`
- `rewind`
- `road`
- `saxophone`
- `scrub-top`
- `share`
- `snare-drum`
- `texting`
- `thought`
- `time-off`
- `trombone`
- `tuba`
- `vacation`
- `wave`
- `xylophone`


## Added the above icons to their respective categories:
### Recently added

- All of the above icons. Icons that were previously in the "Recently Added" category have been replaced with the above icons.

### Academics and Research

- `quotes`

 
### Campus Buildings

- `pappajohn`

 
### Entertainment and Dining

- `band-hat`
- `bass-drum`
- `clarinet`
- `conducting`
- `drum-major`
- `piano`
- `saxophone`
- `snare-drum`
- `trombone`
- `tuba`
- `xylophone`

### Home and Personal
- `ballot`
- `emotions`
- `person`
- `thought`
- `vacation`
- `wave`
 
### Patient Care
- `nurse-cap`
- `scrub-top`
 
### Technology and Business

- `benefits`
- `empty-battery`
- `fast-forward`
- `full-battery`
- `half-battery`
- `idea`
- `pause`
- `play`
- `rewind`
- `share`
- `texting`
- `time-off`

### Wayfinding and Transportation
- `bionic-bus`
- `road`

## Icon adjustments

- Renamed the `doctor`-related icons to `healthcare-professional-*`
- Added `nurse` keyword to `healthcare-professional-*` icons
- Fixed a typo in the `stethoscope` icon name
- Fixed a typo in the `social-distance` icon in the "Patient Care" category
- Adjusted some tags/keywords related to the Hawkeye Wave. Searching for the following terms should get you to Hawkeye Wave-related icons including the Stead Family Children's Hospital, a hand waving, and Kinnick:
    - `wave`,
    - `the wave`
    - `hawkeye wave`
    - `first quarter wave`
    - `the hawkeye wave`
- Added search results for these search terms that previously returned no results: 
    - `medical bill`
    - `handwriting`
    - `survey`
    - `thank you`
    - `thank`
    - `thanks`
    - `survey`
    - `checklist`
    - `questions`

## Icon Browser general updates
 - Code formatting on `top-menu.scss`
 - Bumped node package versions
